md.py

```diff
@@ -154,10 +159,11 @@ def setup_from_args(self, args=None):
                             help="md chunk size")
 
         group = parser.add_mutually_exclusive_group()
-        group.add_argument("-d", "--disks", "--ram-disks", type=int,
+        group.add_argument("-d", "--disks", type=int, default=3,
```

Comment:
It might be better to no set the default explicitly here and just let it be handled below like you have.

Reply:
Most tests, leave this set to it's default value